vips.go

```diff
@@ -51,9 +51,9 @@ type vipsImage struct {
 
 // vipsSaveOptions represents the internal option used to talk with libvips.
 type vipsSaveOptions struct {
-	Speed          int
 	Quality        int
 	Compression    int
+	Speed          int // Speed defines the AVIF encoders CPU effort. Valid values are 0-8.
```

Comment:
Could you limit the type to the expected values 0-8 here?

Reply:
No, Go doesn't have anything like that.